### DISCUSSION TITLE
Garage door

### DISCUSSION THREAD

**Discussion by walnow78** [RavenSystem/esp-homekit-devices#1514] [MEPLHAA Scripts]
I have an issue with my configuration for the garage door. The json that Im using is this one:

{
"c":{"l":13,"b":[{"g":0,"t":5}]},
"a":[{
"t":40,
"d":18,
"0":{"r":[{"g":12,"v":1,"i":0.5}]},
"1":{"a":0},
"2":{"r":[{"g":12,"i":1.5},{"g":12,"v":1,"i":0.5},{"g":12,"v":1,"i":2}]},
"3":{"a":2},
"f3":[{"g":14,"t":0}],
"f4":[{"g":14,"t":1}],
"f2":[{"g":4,"t":0}],
"f5":[{"g":4,"t":1}]
}]
}

Im using two sensors (when the door is open and when the door is close)

The sensors are working fine, I can see the for status open, opening, close and closing. Also I can close the door when the door is open but the issue is when the door is close, I cant open the door. Im getting the notification but nothing happens. The action 0 and 1 is the same, so Im completely lost why is closing the door but is not opening.

Thanks

**Comment by RavenSystem** [comment]
It is very strange. I tested your JSON and works fine. Can you post logs when door is closed and you send "open" command from HomeKit?

**Comment by ceg117** [comment]
I checked your json and found that you have not set the operation definition of opening the door when the door is closed.

**Comment by ceg117** [comment]
I checked your json and found that you have not set the operation definition of opening the door when the door is closed.
